docs/content/https/acme.md

```diff
@@ -8,6 +8,45 @@ You can configure Traefik to use an ACME provider (like Let's Encrypt) for autom
 !!! warning "Let's Encrypt and Rate Limiting"
     Note that Let's Encrypt API has [rate limiting](https://letsencrypt.org/docs/rate-limits).
 
+    Use Let's Encrypt staging server with the [`caServer`](#caserver) configuration directives
+    when experimenting to avoid hitting this limit too fast.
+    
+## Certificate Resolvers
+
+Traefik requires you to define "Certificate Resolvers" in the [static configuration](../getting-started/configuration-overview.md#the-static-configuration), 
+which are responsible for retrieving certificates from an ACME server.
+
+Then, each ["router"](../routing/routers/index.md) is configured to enable TLS,
+and is associated to a certificate resolver through the [`tls.certresolver` configuration directive](../routing/routers/index.md#certresolver).
```

Comment:
```suggestion
and is associated to a certificate resolver through the [`tls.certresolver` configuration option](../routing/routers/index.md#certresolver).
```

Reply:
Challenging this suggestion: looks fine, but how do we deal with the potential confusion with `tls.options` option? I initially wrote option as you suggests, but felt that `directives` was a mor non-ambiguous wording in this context (TLS). WDYT?

Reply:
We will keep `option` to be homogeneous with the other documentation